PostHog/SwiftUI/PostHogTagViewModifier.swift

```diff
@@ -175,7 +184,15 @@
 
         return commonAncestor
             .allDescendants(between: anchorView, and: taggerView)
-            .filter { !$0.postHogView } // exclude injected views
+            .lazy
```

Comment:
what does lazy here do? we did not need before

Reply:
Oh nice, TIL, your call here.
The && is more readable but less functional so you can decide :)